latest/index.bs

```diff
@@ -205,6 +205,18 @@ Metadata {#metadata}
 The various `.zattrs` files throughout the above array hierarchy may contain metadata
 keys as specified below for discovering certain types of data, especially images.
 
+"axes" metadata {#axes-md}
+--------------------------
+
+Describes axes of a physical coordinate space. It is a dictionary, which MUST contain the fields:
+- "labels": list of strings that specify the name per dimension. The values MUST be unique.
+- "types": list of strings that specify the type per dimension. The values SHOULD be one of "space", "channel", "time".
+- "units": list of strings that specify the unit per dimension.
```

Comment:
What are the enumerated values here? e.g. "micrometer", "MICROMETER", "µm"?
In OMERO we use enums `'ANGSTROM', 'ASTRONOMICALUNIT', 'ATTOMETER', 'CENTIMETER', 'DECAMETER', 'DECIMETER', 'EXAMETER', 'FEMTOMETER', 'FOOT', 'GIGAMETER', 'HECTOMETER', 'INCH', 'KILOMETER', 'LIGHTYEAR', 'LINE', 'MEGAMETER', 'METER', 'MICROMETER', 'MILE', 'MILLIMETER', 'NANOMETER', 'PARSEC', 'PETAMETER', 'PICOMETER', 'PIXEL', 'POINT', 'REFERENCEFRAME', 'TERAMETER', 'THOU', 'YARD', 'YOCTOMETER', 'YOTTAMETER', 'ZEPTOMETER', 'ZETTAMETER'` for length and 

`'ATTOSECOND', 'CENTISECOND', 'DAY', 'DECASECOND', 'DECISECOND', 'EXASECOND', 'FEMTOSECOND', 'GIGASECOND', 'HECTOSECOND', 'HOUR', 'KILOSECOND', 'MEGASECOND', 'MICROSECOND', 'MILLISECOND', 'MINUTE', 'NANOSECOND', 'PETASECOND', 'PICOSECOND', 'SECOND', 'TERASECOND', 'YOCTOSECOND', 'YOTTASECOND', 'ZEPTOSECOND', 'ZETTASECOND'` for time.
Although I don't know what the original source for these was.

Reply:
Good point, @will-moore. I have listed this under open questions here: https://github.com/ome/ngff/pull/57#issuecomment-932055668.